src/files.ts

```diff
@@ -132,25 +135,22 @@ export function determineBaseDir(paths: string[]): string {
   return '';
 }
 
-function searchFiles(
+async function* searchFiles(
```

Comment:
I learn something new everyday function* cool thanks 😃

Reply:
It'a generator: instead of returning a value or a promise itself, it returns an async generator so you can then use `for (await value of results)...`